packages/engine/src/world/SketchManager.ts

```diff
@@ -35,14 +35,21 @@ export class SketchManager {
   }
 
   public removeSketchFromScene = (instanceId: string): void => {
-    const scene = getDebugScene().scene
-    const oldSketch = scene.getObjectByName(instanceId)
+    const engineScene = getDebugScene()
+    const threeScene = engineScene.scene
+    const oldSketch = threeScene.getObjectByName(instanceId)
 
     if (!oldSketch) {
+      const pass = engineScene.getPassesByName(instanceId)
```

Comment:
Do we need this method? Can we not just do `this.sketchInstances[instanceId].getPasses`? Maybe you have a good reason for doing it this way though, curious to know.